src/docs/08-operation-guide/index.md

```diff
@@ -0,0 +1,15 @@
+---
+layout: default
+title: Operation Guide
+permalink: /docs/operation-guide
+---
+
+# Operation Guide
+
+## Overview
+
+This document will cover things that you need to know to run a Cadence cluster in production. Topics including: setup, monitoring, maintenance and troubleshooting.
+
+## Cluster Setup and Configuration
+
+- [Cluster Setup and Configuration](01-setup/).
```

Comment:
You can consider to drop this section now as the navigation should be working now.